### PR TITLE
[zh] improve check-inject multicluster

### PR DESCRIPTION
**Pull request by my-git9**
Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/zh/docs/ops/diagnostic-tools/multicluster/index.md [hunk 1]
```diff
@@ -22,17 +24,23 @@ Hello version: v1, instance: helloworld-v1-578dd69f69-j69pf
...
+如果集群分布于不同地理位置（地区/区域），本地负载均衡将优先选用本地实例提供服务，
+这与预期相符。而如果禁用了本地负载均衡或者是集群处于同一地理位置，那就可能还存在其他问题。
 
-### 受信配置{#trust-configuration}
+### 受信配置  {#trust-configuration}
 
-与集群内通信一样，跨集群通信依赖于代理之间公共的且可信任的根证书颁发机构(root)。默认情况下 Istio 使用自身单独生成的根证书颁发机构。对于多集群的情况，我们必须手动配置公共的且可信任的根证书颁发机构。阅读下面的 **Plug-in Certs** 章节或者参考[身份和信任模型](/zh/docs/ops/deployment/deployment-models/#identity-and-trust-models)了解更多细节。
+与集群内通信一样，跨集群通信依赖于代理之间公共的且可信任的根证书颁发机构（root）。
+默认情况下 Istio 使用自身单独生成的根证书颁发机构。对于多集群的情况，
+我们必须手动配置公共的且可信任的根证书颁发机构。阅读下面的 **Plug-in Certs** 章节或者参考
```
Comment (windsonsea) [inline content/zh/docs/ops/diagnostic-tools/multicluster/index.md:42]:
```suggestion
我们必须手动配置公共的且可信任的根证书颁发机构。阅读下面的 **插件证书** 内容或者参考
```

File: content/zh/docs/ops/diagnostic-tools/multicluster/index.md [hunk 2]
```diff
@@ -22,17 +24,23 @@ Hello version: v1, instance: helloworld-v1-578dd69f69-j69pf
...
+[身份和信任模型](/zh/docs/ops/deployment/deployment-models/#identity-and-trust-models)了解更多细节。
 
 **证书插件**：
```
Comment (windsonsea) [inline content/zh/docs/ops/diagnostic-tools/multicluster/index.md:45]:
```suggestion
**插件插件**：
```

File: content/zh/docs/ops/diagnostic-tools/multicluster/index.md [hunk 3]
```diff
@@ -118,26 +129,29 @@ $ istioctl --context $CTX_CLUSTER1 proxy-config endpoint sleep-dd98b5f48-djwdw.s
 10.0.6.13:5000                   HEALTHY     OK                outbound|5000||helloworld.sample.svc.cluster.local
 {{< /text >}}
 
-多网络中模型中，我们期望其中一个端点 IP 与从集群的东西向网关（east-west gateway）公网 IP 匹配。看到多个 Pod IP 说明存在以下两种情况：
+多网络中模型中，我们期望其中一个端点 IP 与从集群的东西向网关（east-west gateway）公网 IP 匹配。
+看到多个 Pod IP 说明存在以下两种情况：
 
 * 无法确定远程网络的网关地址。
-* 无法确定客户端 Pod 或服务器 Pod 的网络。
+* 无法确定客户端 Pod 或服务端 Pod 的网络。
```
Comment (windsonsea) [inline content/zh/docs/ops/diagnostic-tools/multicluster/index.md:136]:
```suggestion
* 无法确定客户端 Pod 或服务器端 Pod 的网络。
```
`服务端`和`服务器`不太一样呐

File: content/zh/docs/ops/diagnostic-tools/multicluster/index.md [hunk 4]
```diff
@@ -118,26 +129,29 @@ $ istioctl --context $CTX_CLUSTER1 proxy-config endpoint sleep-dd98b5f48-djwdw.s
...
-如果 `EXTERNAL-IP` 卡在 `<PENDING>` 状态, 则说明环境可能不支持 `LoadBalancer` 服务。在这种情况下，可能需要自定义服务的 `spec.exteralIPs` 部分，手动为网关提供集群外可达的 IP。
+如果 `EXTERNAL-IP` 卡在 `<PENDING>` 状态, 则说明环境可能不支持 `LoadBalancer` 类型的 Service。
+在这种情况下，可能需要自定义 Service 的 `spec.exteralIPs` 部分，手动为网关提供集群外可达的 IP。
 
-如果外部 IP 存在，请检查 `topology.istio.io/network` 标签的值是否正确。如果不正确，请重新安装网关，并确保在生成脚本上设置 --network 标志。
+如果外部 IP 存在，请检查 `topology.istio.io/network` 标签的值是否正确。如果不正确，
+请重新安装网关，并确保在生成脚本上设置 --network 标志。
 
-**无法确定客户端 Pod 或服务器 Pod 的网络**：
+**无法确定客户端 Pod 或服务端 Pod 的网络**：
```
Comment (windsonsea) [inline content/zh/docs/ops/diagnostic-tools/multicluster/index.md:154]:
```suggestion
**无法确定客户端 Pod 或服务器端 Pod 的网络**：
```
同上，server pod 翻成`服务端`有点问题

File: content/zh/docs/ops/diagnostic-tools/multicluster/index.md (hunk 2 above)
Comment (windsonsea) [reply content/zh/docs/ops/diagnostic-tools/multicluster/index.md:45]: 啊额，原来我打错字了 -_! 这就是上文中提到的“插件证书”

Comment (zirain) [reply content/zh/docs/ops/diagnostic-tools/multicluster/index.md:45]: 如果原文是 `Plug-in`，翻译成 `插入式` 更好？

**Review by windsonsea — APPROVED** [review]
Thanks. LGTM.

**Pull request merged**